junit.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Let's add this to git ignore.

Reply:
We still have to ditch this file. I'm not sure if we need to delete it then push that then ignore it?

Reply:
I see what you are saying.. I did not realize that this has all the last test run information. I think I will have to delete it and push the changes and ignore the file in gitignore. Thank you for catching it.